test/Parse/interpolated_trailing_quote.swift

```diff
@@ -0,0 +1,6 @@
+// Check the trailingQuoteLoc field of InterpolatedStringLiteral
+
+"\("abc")"
+
+// RUN: %target-swift-frontend -dump-ast %s | %FileCheck %s
+// CHECK: (interpolated_string_literal_expr {{.*}} trailing_quote_loc={{.*}}:3:10 {{.*}}
```

Comment:
I don't think we actually test where the AST puts specific SourceLocs in any systematic way, but maybe we should...

Reply:
Yes, but out of scope.

Reply:
Definitely. I'm just thinking that maybe the filename and test setup should be designed to allow for other location tests—move it to `test/Parse/source_locs.swift` and perhaps wrap the statement in a `func string_interpolation() { ... }` to make it easier to distinguish from anything else that we might add.

Reply:
Good idea, done!